src/ReefModEngine.jl

```diff
@@ -23,6 +23,19 @@ macro RME(func)
     end
 end
 
+"""
+Only for use when RME functions return non-error numeric results. 
+
+# Examples
+
+```julia
+count::Float64 = @getRME ivOutplantCountPerM2("iv_name"::Cstring)::Cdouble
```

Comment:
One more - update example variable as `count` is also a function.

```suggestion
count_per_m2::Float64 = @getRME ivOutplantCountPerM2("iv_name"::Cstring)::Cdouble
```